components/authentication-framework/org.wso2.carbon.identity.application.authentication.framework/src/main/java/org/wso2/carbon/identity/application/authentication/framework/config/model/graph/graaljs/GraalSerializableJsFunction.java

```diff
@@ -75,12 +75,8 @@ public void setFunction(boolean function) {
     public Object apply(Context polyglotContext, Object... params) {
 
         if (isPolyglotFunction) {
-            try {
-                Value jsFunction = polyglotContext.eval("js", "(" + getSource() + ")");
-                return jsFunction.execute(params);
-            } catch (PolyglotException e) {
-                log.error("Error when executing function", e);
```

Comment:
Is there an error log containing the details at another level?

Reply:
yes at https://github.com/wso2/carbon-identity-framework/blob/20ee62e45d15cfecbebedfed67cd55e21300ffc6/components/authentication-framework/org.wso2.carbon.identity.application.authentication.framework/src/main/java/org/wso2/carbon/identity/application/authentication/framework/config/model/graph/graaljs/JsGraalGraphBuilder.java#L648